README.md

```diff
@@ -69,6 +81,49 @@ Okta's Spring Security integration will [parse the JWT access token](https://dev
 Check out a minimal example that uses the [Okta Signin Widget and JQuery](examples/siw-jquery) or [this blog post](https://scotch.io/tutorials/build-a-secure-notes-application-with-kotlin-typescript-and-okta). 
 
 
+## Supporting server side applications - OAuth2 Code flow
+
+Building a server side application and just need to redirect to a login page? This OAuth2 code flow is for you.
```

Comment:
I think we agreed it should be either "OAuth" or "OAuth 2.0"

Reply:
+1 for `OAuth 2.0` that is what we have on oauth.net, i'll update